app/Workflow.php

```diff
@@ -60,7 +60,7 @@ public function verificationComments()
     }
 
     public function search($title){
-        return Workflow::join('users', 'users.id', '=', 'author_id')->where('title', 'LIKE', '%'.$title.'%')->orWhere('description', 'LIKE', '%'.$title.'%')->select('users.id','users.name AS name', 'workflows.*')->get();
+        return Workflow::join('users', 'users.id', '=', 'author_id')->where('title', 'LIKE', '%'.$title.'%')->orWhere('description', 'LIKE', '%'.$title.'%')->select('users.id','users.first_name', 'users.last_name', 'workflows.*')->get();
```

Comment:
I think it would be better to use more eloquent-like approach, so you would only need to leave 'where' clause. You can then access the author of the workflow and his/her fields by $workflow->author().

Reply:
Patient side, cannot comment on it for now